tests/tests_pytorch/utilities/test_compile.py

```diff
@@ -17,29 +17,16 @@
 import torch
 from lightning_utilities.core import module_available
 
-from lightning.fabric.utilities.imports import _TORCH_GREATER_EQUAL_2_1
 from lightning.pytorch import LightningModule, Trainer
 from lightning.pytorch.demos.boring_classes import BoringModel
 from lightning.pytorch.utilities.compile import from_compiled, to_uncompiled
 from tests_pytorch.conftest import mock_cuda_count
 from tests_pytorch.helpers.runif import RunIf
 
 
-def skip_if_unsupported():
-    if _TORCH_GREATER_EQUAL_2_1:
-        from torch._dynamo.eval_frame import is_dynamo_supported
-
-        if not is_dynamo_supported():
-            pytest.skip("TorchDynamo unsupported")
-    elif sys.platform == "win32" or sys.version_info >= (3, 11):
-        pytest.skip("TorchDynamo unsupported")
-
-
-@RunIf(min_torch="2.0.0")
+@RunIf(min_torch="2.0.0", dynamo=True)
 @pytest.mark.skipif(sys.platform == "darwin", reason="https://github.com/pytorch/pytorch/issues/95708")
```

Comment:
We could include the min_torch 2.0.0 and darwin platform check inside `dynamo=True` so that they don't need to be set separately

Reply:
the special skipif does not apply to all tests so I didn't include it